test/test_framework_test.go

```diff
@@ -735,7 +735,7 @@ func TestImportContract(t *testing.T) {
                 Test.assert(blockHeight > 1)
 
                 mintFlow(to: admin, amount: 500.0)
-                Test.assertEqual(500.0, getFlowBalance(account: admin))
+                Test.assertEqual(500.001, getFlowBalance(account: admin))
```

Comment:
I feel these small value differences in account balances could be annoying for users when writing tests.
(Also not sure why it changes the account balances when the storage limit is enabled).

So maybe we should have this disabled by default (to make the simple use-cases simple), and allow developers to enable it only if they need it.

Reply:
> I feel these small value differences in account balances could be annoying for users when writing tests.

That' a good point :+1: 

> (Also not sure why it changes the account balances when the storage limit is enabled).

The `emulator.WithStorageLimitEnabled(true)` means that every account has to be created with the minimum storage reservation (`0.001` FLOW), otherwise the account would break the storage limit constraint.

> So maybe we should have this disabled by default (to make the simple use-cases simple), and allow developers to enable it only if they need it.

Sounds good, I will try the approach where this is enabled per test file, with a pragma directive (`#storageLimitEnabled`).